### PR TITLE
added rating

### PR DESCRIPTION
**Pull request by Fcesr**
added star rates to Restaurants detail screen

### CODE REVIEW

**Review by jonathans199 — CHANGES_REQUESTED** [review]
It looks great, however, lets have the background match the background of the app, or wrap it on its own background with a border

simulator_screenshot_5670390A-DC27-4196-A014-3FD5F4A8D2C2

File: screens/ResDetailScreen.js [hunk 1]
```diff
@@ -16,7 +16,9 @@ function ResDetailScreen(props) {
       body: JSON.stringify({ rating: rating }),
     })
       .then(response => response.json())
-      .then(() => props.navigation.navigate('Home'))
+      .then(() => {
+        props.navigation.navigate('Home')
```
Comment (jonathans199) [inline screens/ResDetailScreen.js]: Lets make this one linner, great job @nicole-allen679

File: screens/ResDetailScreen.js [hunk 2]
```diff
@@ -28,12 +30,8 @@ function ResDetailScreen(props) {
...
       </View>
 
-      <Input
-        placeholder='Rate here'
-        style={{ width: 200, height: 60 }}
-        keyboardType='numeric'
-        onChangeText={e => setRating(e)}
-      />
+      <Rating showRating ratingCounting="{5}" onFinishRating={(rating) => setRating(rating)} />
+
```
Comment (jonathans199) [inline screens/ResDetailScreen.js]: Lets get rid of this extra space @nicole-allen679

**Comment by Fcesr** [comment]
added margin and cleaned code

**Pull request merged**